core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -47,8 +47,11 @@ class PolicyManager {
     }
 
     fun addPolicyToTransients(policy: Policy) {
-        for (unique in policy.uniqueObjects)
-            policyUniques.addUnique(unique)
+        for (unique in policy.uniqueObjects) {
+            // Should be replaced with a conditional of the same form later
+            if (!unique.text.contains(Regex("for \\[[0-9]*\\] turns")))
```

Comment:
Please make this a static const

Reply:
The regex that is
Making the regex every time we add a policy is very inefficient

Reply:
True, Regex objects _can_ and in any case with reuse _should_ have a compiled form of the pattern.
As for the pattern itself - I would recommend standardizing on a "transport" form throughout a project, meaning its presentation to the compiler. I'm slightly in favour of using const val with a raw string everywhere, despite kotlin's raw strings being severely broken. const val would be to enable debugger inspection to make sure the pattern actually contains exactly what's intended.
Lastly, that pattern matches ""for [] turns", probably slightly unintended. `?` = 0 or 1, `+` = 1..∞, `*` = 0..∞, `{n}` = exactly n, `{n,m}` = n..m repetitions...